src/CoreWCF.Primitives/src/CoreWCF/Dispatcher/DataContractSerializerOperationFormatter.cs

```diff
@@ -564,7 +564,7 @@ private object DeserializeParameterPart(XmlDictionaryReader reader, PartInfo par
             return val;
         }
 
-        internal static Type GetSubstituteDataContractType(Type type, out bool isQueryable)
+        public static Type GetSubstituteDataContractType(Type type, out bool isQueryable)
         {
```

Comment:
This is a static helper method. Leave internal and duplicate in WebHttp

Reply:
This is addressed in 56eb7b2cffb98978a8ccc6c78635c4672936c041